examples/block_jid.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python3
```

Comment:
This will break in virtualenv's; if this is necessary set it to `/usr/bin/env python3` to ensure the correct version (global or the one in the virtualenv) is used.